ql/experimental/callablebonds/treecallablebondengine.hpp

```diff
@@ -52,6 +52,7 @@ namespace QuantLib {
                                                  Handle<YieldTermStructure>()) ;
         //@}
         void calculate() const;
+        void calculateSpread(double s) const;        
```

Comment:
`calculateWithSpread` is probably better (you're not calculating the spread).

It might also be made `private`.